ql/src/java/org/apache/hadoop/hive/ql/io/CombineHiveRecordReader.java

```diff
@@ -113,7 +113,9 @@ private PartitionDesc extractSinglePartSpec(CombineHiveInputSplit hsplit) throws
     for (Path path : hsplit.getPaths()) {
       PartitionDesc otherPart = HiveFileFormatUtils.getFromPathRecursively(
           pathToPartInfo, path, cache);
-      LOG.debug("Found spec for " + path + " " + otherPart + " from " + pathToPartInfo);
+      if (LOG.isDebugEnabled()) {
```

Comment:
This check isn't needed as you are using "`LOG.debug("Found spec for {} {} from {}", path, otherPart, pathToPartInfo);`" anyways. They get evaluated lazily. 

+1.

Reply:
I spent a lot of time trying to scrub this behavior from the code.  Please do not include the logging guards and just use the anchors `{}`

Reply:
okay, thanks @rbalamohan and @belugabehr  for the review
I'm removing the guards and then I'm going to merge